store/src/java/com/zimbra/cs/mailbox/ContactAutoComplete.java

```diff
@@ -764,14 +764,16 @@ public void addMatchedContacts(String query, Map<String, ? extends Object> attrs
         Mailbox mbox = MailboxManager.getInstance().getMailboxByAccountId(getRequestedAcctId());
         if (folderIDs == null) {
             for (Folder folder : mbox.getFolderList(octxt, SortBy.NONE)) {
-                if (folder.getDefaultView() != MailItem.Type.CONTACT || folder.inTrash()) {
-                    continue;
-                } else if (folder instanceof Mountpoint) {
-                    Mountpoint mp = (Mountpoint) folder;
+            	//added an extra condition in if block (ZBUG-2588)
+            	//it was never going into the Mountpoint condition
+            	if (folder instanceof Mountpoint) {
+            		Mountpoint mp = (Mountpoint) folder;
```

Comment:
Use not a tab but a white space for indent.

Reply:
sure I will do